server/apps/main/forms.py

```diff
@@ -63,6 +63,20 @@ class ShareExperienceForm(forms.Form):
     moderation_status = forms.ChoiceField(choices = statuses, widget = forms.Select(), required=False)
     moderation_status.group = "hidden"
 
+    authorship_choices = [
+        (True, "Experience is my own"),
+        (False, "Experience is someone else's"),
+    ]
+    first_hand_authorship = forms.ChoiceField(choices = authorship_choices, widget = forms.RadioSelect(), required=False)
```

Comment:
I think this should actually be `required=True`, as otherwise it will crash as it allows for submitting the form without a choice (which is `""`) and then crashes as the model doesn't validate that: 

```
ValidationError at /main/share_exp/

['“” value must be either True or False.']

 /code/server/apps/main/views.py, line 173, in share_experience 
```

Reply:
Actioned in [6cb302d](https://github.com/alan-turing-institute/AutSPACEs/pull/580/commits/6cb302d57d5b05def4bf729a7fb85487a9095e9c)

Reply:
Actually, I think this was accidentally reverted again somewhere!